localization/languages/en-US.json

```diff
@@ -157,6 +157,12 @@
         "customSearchEngineDescription": "Replace the search term with %s",
         "settingsKeyboardShortcutsHeading": "Keyboard Shortcuts",
         "settingsKeyboardShortcutsHelp": "Use commas to separate multiple shortcuts.",
+        "settingsProxyHeading": "Proxy",
+        "settingsNoProxy": "No Proxy",
```

Comment:
If no proxy is configured in Min, but there's a proxy configured at the OS level, the OS proxy would still be used, right? If so, this should probably be something like "Use system settings".

Reply:
With these changes, if No Proxy is selected, Min doesn't use a proxy.